docs/02-for-app-authors/03-appdata-guidelines/01-quality-guidelines.md

```diff
@@ -69,8 +69,62 @@ The summary should ideally be no longer than 35 characters, and must be shorter
 
 ### Not technical
 
-The summary should be understandable by a non-technical person, and not contain any technical terms (e.g. `Chat app written in Rust and GTK4/Libadwaita`). Don't mention the toolkit, programming language, or other implementation details that would not be relevant to a non-developer and explain what the app does for people instead.
+The summary should be understandable by a non-technical person, and not contain any technical terms (e.g. "Chat app written in Rust and GTK4/Libadwaita"). Don't mention the toolkit, programming language, or other implementation details that would not be relevant to a non-developer and explain what the app does for people instead.
 
 ### No weird formatting
 
 The summary should not have any weird formatting or punctuation. It shouldn't end with a full stop.
+
+## Description
+
+### Don't repeat the summary
+
+The description should not just repeat or rephrase the summary. The summary is a kind of slogan or advertisement for the app, while the description should go into more detail about the purpose of the app, which features it has, and what makes it unique.
+
+### Not too short
+
+The description should be substantially longer than the summary. Something like 3-6 lines of text at default line widths (around 70 characters) is a good length for most apps, for very large apps up to about 10 lines.
+
+### No super long lists
+
+Avoid feature lists with more than 10 items, they are difficult to scan and make the app store listing unwieldy. Instead, try to group points or use paragraphs instead of bullet points.
+
+## Screenshots
```

Comment:
Seem to miss captions? They are not used right now, but hopefully will be when libappstream lands.